deployment/distribution/build.gradle.kts

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright (c) 2018 Codice Foundation
+
+Released under the GNU Lesser General Public License; see
+http://www.gnu.org/licenses/lgpl.html
+*/
+// Build file
+apply(plugin = "application")
+
+description = "Script and CLI to run the tests."
+
+group = "org.codice.samlconf.deployment"
+
+configure<ApplicationPluginConvention> {
+    applicationName = "samlconf"
+    mainClassName = "org.codice.ctk.CommandKt"
+
+    applicationDefaultJvmArgs = listOf("-Dapp.home=SAMLCTK_APP_HOME")
+
+    applicationDistribution.from("src/main/resources/") {
+        into( "conf")
+    }
+
+    // Copy implementation examples into the distribution
+    project(":external:implementations").getSubprojects().forEach { subProject ->
+        applicationDistribution
+                .into("implementations/" +
+                        subProject.name.replace("samlconf-", "")
+                                .replace("-impl", "")) {
+            from (subProject.file("build/resources/main"))
+            from (subProject.file("build/libs"))
+        }
+    }
+}
+
+dependencies {
+    compile(project(":external:implementations:samlconf-ddf-impl"))
+    compile(project(":external:implementations:samlconf-keycloak-impl"))
+    compile(project(":library"))
+    compile(project(":external:samlconf-plugins-api"))
+    compile(project(":ctk:common"))
+    compile(project(":ctk:idp"))
+
+    compile(Libs.kopperTyped)
+    compile(Libs.jansi)
+}
+
+// We don"t want to include the implementation jars in the classpath
+// since they"re loaded from a specific directory.
+tasks {
+    "copyToLib"(Copy::class) {
+        from(configurations.runtime.exclude(group = "org.codice.samlconf.implementations"))
```

Comment:
This should be correct now. It does exclude the implementation jars and, transitively, `metainf-services-1.1.jar` and `gson-2.8.0.jar`. I did notice that because of that latter transitive exclusion that an older version of the gson jar - `gson-1.4.jar` - gets pulled into the lib directory.

This is the same behavior as on master.